docs/source/man.rst

```diff
@@ -642,6 +642,10 @@ The ``qpc insights upload`` command can be used to upload an insights report to
 
   Contains the report identifier to use to retrieve and upload the insights report. Mutually exclusive with the ``--scan-job`` option.
 
+``--input-file=path to tar.gz containing the Insights report``
```

Comment:
you need to add the input file to the top level `qpc insights upload command`

Reply:
Done.  Good catch.